src/pint/toa.py

```diff
@@ -947,6 +948,25 @@ def get_freq_array(bfv, ntoas):
 
 
 def _group_by_gaps(t, gap):
+    """
+    A utility function to group times according to gap-less stretches.
```

Comment:
First line of a docstring should go on the first line so it shows up as a short description in the docs. But this function is private so a docstring is less important.